website/docs/r/cloud_project_database_kafka_acl.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "ovh"
+page_title: "OVH: cloud_project_database_kafka_acl"
+sidebar_current: "docs-ovh-resource-cloud-project-database-kafka-acl"
+description: |-
+  Creates an acl for a kafka cluster associated with a public cloud project.
+---
+
+# ovh_cloud_project_database_kafka_acl
+
+Creates an acl for a kafka cluster associated with a public cloud project.
+
+## Example Usage
+
+```hcl
+data "ovh_cloud_project_database" "kafka" {
+  service_name  = "XXX"
+  engine        = "kafka"
+  cluster_id    = "ZZZ"
+}
+
+resource "ovh_cloud_project_database_kafka_acl" "acl" {
```

Comment:
The example does not match with the definition in the code.
For example, `permission` is required, and also `topic` and `username`.